lib/ridgepole/ext/abstract_mysql_adapter/use_alter_index.rb

```diff
@@ -6,16 +6,38 @@ module Ridgepole
   module Ext
     module AbstractMysqlAdapter
       module UseAlterIndex
-        def add_index(table_name, column_name, options = {})
-          index_name, index_type, index_columns, index_options, _index_algorithm, index_using = add_index_options(table_name, column_name, **options)
+        if ActiveRecord.gem_version >= Gem::Version.new('6.1.0')
```

Comment:
I think it would be better to remove the `--mysql-use-alter` option.
This option was added to `alter add index` to add an algorithm_option/lock_option (`LOCK=NONE, ALGORITHM=INPLACE`).
However, `CREATE INDEX` can also add algorithm_option/lock_option, so modify the `--alter-extra` option.

(Please wait a moment while I create a pull request.)

Reply:
fixed https://github.com/winebarrel/ridgepole/pull/330